### PR TITLE
Adding access to root ref

### PR DESCRIPTION
**Pull request by kserjey**
**What**:

Related to #743 

**Why**:

To get access to root element

**How**:

Calling ref with `callAll` as it done for `getMenuProps`

**Checklist**:

- [ ] Documentation N/A
- [x] Tests
- [x] Ready to be merged
      
- [ ] Added myself to contributors table
      

`npm run add-contributor` doesn't work, it throws an error 🤷‍♀

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -1063,6 +1063,18 @@
...
+      "profile": "https://github.com/kserjey",
+      "contributions": [
+        "bug",
+        "code",
+        "test"
+      ]
     }
-  ]
+  ],
+  "commitConvention": "none"
```
Comment (silviuavram) [inline .all-contributorsrc]: I don't think this is needed.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ autocomplete/dropdown/select/combobox components</p>
 [![downloads][downloads-badge]][npmcharts] [![version][version-badge]][package]
 [![MIT License][license-badge]][license]
 
-[![All Contributors](https://img.shields.io/badge/all_contributors-108-orange.svg?style=flat-square)](#contributors)
+[![All Contributors](https://img.shields.io/badge/all_contributors-109-orange.svg?style=flat-square)](#contributors-)
```
Comment (silviuavram) [inline README.md]: link is broken now.

File: README.md [hunk 3]
```diff
@@ -164,9 +164,9 @@ const items = [
 
 render(
   <Downshift
-    onChange={selection => alert(
-      selection ? `You selected ${selection.value}` : 'Selection Cleared'
-    )}
+    onChange={selection =>
```
Comment (silviuavram) [inline README.md]: not needed change

**Comment by kserjey** [comment]
I tried to install kcd-scripts globally, but with no success. As far as I found, there is no `contributors.js` in `kcd-scripts/src/scripts`. Any ideas? And what do you mean when you said remove commit? `git reset + git push --force` or simply `git revert`?

File: README.md (hunk 3 above)
Comment (kserjey) [reply README.md]: I think it was made by `prettier`. I can temporary remove `lint-staged` if you want.

**Comment by silviuavram** [comment]
Anyway I think overall it's ok. I made 3 comments on the PR, correct them (apart from the prettier related one) and we should be good to go.

Also re-generate the size-snapshot and override it to resolve the conflict.

Looking forward to merge this change soon. Thank you!

**Comment by kserjey** [comment]
Sorry for the late response. Fixed your comments! I'm not sure about size-snapshot, I just run `npm run build` and committed changes. Am I right?

**Comment by silviuavram** [comment]
https://github.com/downshift-js/downshift/pull/754 will fix the build. we will merge it here, re-trigger and merge afterwards.

**Comment by kserjey** [comment]
Cool! 👍

**Comment by silviuavram** [comment]
@kserjey can you merge/rebase with current master? Let's see if the build passes.

**Comment by silviuavram** [comment]
https://github.com/downshift-js/downshift/pull/756

**Pull request closed without merging**